docs/guides/syntax.md

```diff
@@ -81,43 +81,43 @@ Optional.
 
 A suffix mapped to a [pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes), for example:
 
-| Pseudo-Element       | Suffix  |
-| -------------------- | ------- |
-| `:active`            | `:a`    |
-| `:checked`           | `:c`    |
-| `:default`           | `:d`    |
-| `:disabled`          | `:di`   |
-| `:empty`             | `:e`    |
-| `:enabled`           | `:en`   |
-| `:first`             | `:fi`   |
-| `:first-child`       | `:fc`   |
-| `:first-of-type`     | `:fot`  |
-| `:fullscreen`        | `:fs`   |
-| `:focus`             | `:f`    |
-| `:focus-within`      | `:fw`   |
-| `:focus-visible`     | `:fv`   |
-| `:hover`             | `:h`    |
-| `:indeterminate`     | `:ind`  |
-| `:in-range`          | `:ir`   |
-| `:invalid`           | `:inv`  |
-| `:last-child`        | `:lc`   |
-| `:last-of-type`      | `:lot`  |
-| `:left`              | `:l`    |
-| `:link`              | `:li`   |
-| `:only-child`        | `:oc`   |
-| `:only-of-type`      | `:oot`  |
-| `:optional`          | `:o`    |
-| `:out-of-range`      | `:oor`  |
-| `:placeholder-shown` | `:ps`   |
-| `:read-only`         | `:ro`   |
-| `:read-write`        | `:rw`   |
-| `:required`          | `:req`  |
-| `:right`             | `:r`    |
-| `:root`              | `:rt`   |
-| `:scope`             | `:s`    |
-| `:target`            | `:t`    |
-| `:valid`             | `:va`   |
-| `:visited`           | `:vi`   |
+| Pseudo-Element       | Suffix |
```

Comment:
Rebase from `main` will pull in this change for you

Reply:
done